src/charm.lua

```diff
@@ -747,6 +771,10 @@ end
 sendInfoMessage("G.FUNCS.can_discard() patched. Reason: Discarding zero cards", "Bakery")
 
 G.FUNCS.Bakery_discard_zero = function(e)
+    if #G.Bakery_charm_area.cards == 0 or is_charm_debuffed_safe() then
+        return
+    end
```

Comment:
This check is fine, even if you end up reworking the debuff system, although it'd be better to have parity with the check in the hook for `G.FUNCS.can_discard`.

Reply:
Resolved in bbaa626